tensorrt_llm/schedule_params.py

```
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass(slots=True, kw_only=True)
+class ScheduleParams:
+    """Schedule parameters.
+
+    Args:
+        attention_dp_rank (int): The rank of target attention dp
+        attention_dp_relax (bool): Whether to allow the request to be scheduled to other attention dp for better throughput
```

Ruff (0.12.2)

Line too long (123 > 120)

(E501)